src/backend/cache.py

```diff
@@ -52,17 +54,20 @@ def fetch_data():
         headers = headers
     )
 
-    return res
+    return sorted(
```

Comment:
Previously another function sorted it after some processing but doing it here avoids needing to do it again.

Reply:
Also not returning the whole response because we are no longer caching the raw data